doc/rst/source/text_common.rst_

```diff
@@ -226,6 +226,6 @@ Optional Arguments
 
 .. include:: explain_-qi.rst_
 
-.. include:: explain_-tv.rst_
+.. include:: explain_-tv_full.rst_
```

Comment:
Can we remove the file `explain_-tv.rst_` now?

Reply:
yes, otherwise it will not be referenced.  WiIl do.